### DISCUSSION TITLE
Flash Game Support

### DISCUSSION THREAD

**Discussion by Satanarious** [OnionUI/Onion#1009] [General]
There are a couple of Linux Flash players that can run Flash games easily, namely:

1. [Ruffle](https://ruffle.rs/)
2. [Flashpoint](https://flashpointarchive.org/datahub/Linux_Support)

Official support from onionOS would be highly appreciated, because there are thousands of flash games available to play, which can easily run on the miyoo mini/ miyoo mini plus.

Do you guys think this is possible ?

**Comment by stevodotorg** [comment]
Wish I knew how to do it. Would LOVE to see this.

**Comment by schmurtzm** [comment]
I don't think that it is technically possible sadly.

**Comment by michaelachrisco** [comment]
I wonder if Ruffle Integration could happen?

**Reply by Satanarious** [reply]
I tried to run it using the terminal on my miyoo mini plus, but no luck there.

**Reply by michaelachrisco** [reply]
Thanks for trying. I believe in order to get this working, one would have to compile it using https://github.com/ruffle-rs/ruffle?tab=readme-ov-file#linux-prerequisites and one would have to make sure each dependency is supported on the MMP. I tried a bit of it last night...no dice, too many errors. If someone else wants to take up the mantle, go right head :)
